### PR TITLE
add includes for sensitive-data

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/common/data-management/sensitive-data/index.mdx [hunk 1]
```diff
@@ -66,23 +66,15 @@ For more details, see [_Filtering Events_](../../configuration/filtering/).
...
 Your organization may determine that emails are not considered confidential, but if they are, consider instead sending your internal identifier:
 
-```javascript
-Sentry.setUser({ id: user.id });
-
-// or
-
-Sentry.setUser({ username: user.username });
-```
+<PlatformContent includePath="sensitive-data/set-user" />
```
Comment (marandaneto) [inline src/platforms/common/data-management/sensitive-data/index.mdx:77]:
```suggestion
<PlatformContent includePath="set-user" />
```

Comment (marandaneto) [reply src/platforms/common/data-management/sensitive-data/index.mdx:77]: we could use the already defined `set-user` but it's not exactly the same example, wdyt?

File: src/platforms/common/data-management/sensitive-data/index.mdx [hunk 2]
```diff
@@ -66,23 +66,15 @@ For more details, see [_Filtering Events_](../../configuration/filtering/).
 
 Instead of sending confidential information in plaintext, considering hashing it:
 
-```javascript
-Sentry.setTag("birthday", checksum_or_hash("08/12/1990"));
-```
+<PlatformContent includePath="sensitive-data/set-tag" />
```
Comment (marandaneto) [inline src/platforms/common/data-management/sensitive-data/index.mdx:69]: here we could not reuse the already defined `set-tag` cus this example has specific PII data and the existent `set-tag` don't

**Comment by marandaneto** [comment]
how do I make this include fallback to any existing platform? right now if I enter not from Java/Android/JS, it says it's not supported but ideally it'd fall back to any of them.

**Comment by PeloWriter** [comment]
@marandaneto - does this PR fix the fallback issue? https://github.com/getsentry/sentry-docs/pull/2483

File: src/includes/sensitive-data/set-user/java.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+// or
+
+```java
+import io.sentry.Sentry;
+import io.sentry.protocol.User;
+
+Sentry.configureScope(scope -> {
+    User user = new User();
+    user.setUsername(clientUser.getUsername());
+    scope.setUser(user);
```
Comment (bruno-garcia) [inline src/includes/sensitive-data/set-user/java.mdx:21]:
```suggestion
import io.sentry.Sentry;
import io.sentry.protocol.User;

Sentry.configureScope(scope -> {
    User user = new User();
    // User Id:
    user.setId(clientUser.getId());
    // Or Username"
    user.setUsername(clientUser.getUsername());
    scope.setUser(user);
```

File: src/platforms/common/data-management/sensitive-data/index.mdx (hunk 2 above)
Comment (bruno-garcia) [inline src/platforms/common/data-management/sensitive-data/index.mdx:69]: Would this show as non-existent in all other platforms now that doesn't have a sample?

Comment (marandaneto) [reply src/platforms/common/data-management/sensitive-data/index.mdx:69]: yep, that's why I'd like to tackle this https://github.com/getsentry/sentry-docs/pull/2491#issuecomment-705465385

**Comment by marandaneto** [comment]
> @marandaneto - does this PR fix the fallback issue? #2483

Not really, because I'd like to fallback only for this page (include-sensitive-data) till all of us get to add the specific code snippets for all the languages we support as I added only for 2 languages on this PR.
The PR above would fallback for all the pages.

`<PlatformContent includePath="sensitive-data/set-tag" platform="javascript" />` also didn't work

> defaults to the `platform` value from the page context or querystring

this PR already works as it is but for most of the languages, the code snippets are still missing.

File: src/includes/sensitive-data/set-tag/javascript.mdx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+```javascript
+Sentry.setTag("birthday", checksum_or_hash("08/12/1990"));
```
Comment (lobsterkatie) [inline src/includes/sensitive-data/set-tag/javascript.mdx:2]:
```suggestion
Sentry.setTag("birthday", checksumOrHash("08/12/1990"));
```

IDK why it was snakecased in the old example.

**Review by HazAT — APPROVED** [review]
I would merge it like this, it will help us to fill in the missing content

**Pull request merged**